### PR TITLE
[12.0] [ADD] website_sale_stock_picking_note: New module that allow clients to set commets on the website orders and its pickings

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT25254

On next image you have an example of comments input.
image

Please @chienandalu @victoralmau, can you review this?

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Don't we have comments on standard (set via customize_show)? If not, the label should be "Custom delivery comments".

**Comment by chienandalu** [comment]
The idea is to make it compatible with `sale_stock_picking_note` so it would become handy for a new module that will allow internal users to fulfill website orders in behalf of the customers. Additionaly the extra step isn't connected to a real field, so the UX feels clunky when the user goes back and forward and the writting comment disappears. This improves that as well

**Comment by pedrobaeza** [comment]
OK, please explain this motivation in the module description and change the label to what I'm commenting.

**Comment by chienandalu** [comment]
@CarlosRoca13 please do.

**Comment by CarlosRoca13** [comment]
@chienandalu Changes done

File: website_sale_stock_picking_note/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website Sale Stock Picking Note",
+    "summary": "Allows to set comments on website orders",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
+    "version": "12.0.1.0.0",
+    "category": "Website",
+    "depends": ["website_sale", "sale_stock_picking_note"],
```
Comment (victoralmau) [inline website_sale_stock_picking_note/__manifest__.py:11]: Add yourself as a maintainer of this new addon.

**Review by victoralmau — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
@victoralmau Change done

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Functional review. (runbot)

1- I wouldn't make user to click "Set comments" button to record his/her comments. I would always add the content of the "Comments text box". Do you know what I mean?

2- I'm trying to click the "Do you want to disable custom delivery comment feature" but I can't edit the checkbox.

3- I would change the "Do you want to disable custom delivery comment feature" text and leave only "Disable custom delivery comment feature".

**Comment by CarlosRoca13** [comment]
@HaraldPanten I did all changes that you suggest, can you check it again?

**Review by HaraldPanten — APPROVED** [review]
Good job!

Now LGTM, just wait for travis 👍

**Pull request merged**